source/hooks/useCreateTransaction.tsx

```diff
@@ -0,0 +1,38 @@
+import {useState} from 'react';
+import {browser} from 'webextension-polyfill-ts';
+import moneyFormatter from '../utils/moneyFormatter';
+import DASH_SERVICE_MESSAGES from '../Background/services/messages';
+import {TransactionInfo} from '../Background/services/types';
+
+const useCreateTransaction = () => {
+  const [transaction, setTransaction] = useState<TransactionInfo>();
+  const handleCreateTransaction = async (data: {
+    amount: number;
+    address: string;
+  }): Promise<void> => {
+    const options = {
+      satoshis: +moneyFormatter.formatSatoshis(+data.amount).toFixed(),
```

Comment:
I don't think we need .toFixed() here (having in mind that formatSatoshis returns rounded number)